pkg/scheduler/apis/config/validation/validation.go

```diff
@@ -17,17 +17,16 @@ limitations under the License.
 package validation
 
 import (
-	apimachinery "k8s.io/apimachinery/pkg/apis/config/validation"
 	"k8s.io/apimachinery/pkg/util/validation"
 	"k8s.io/apimachinery/pkg/util/validation/field"
-	apiserver "k8s.io/apiserver/pkg/apis/config/validation"
+	componentbase "k8s.io/component-base/config/validation"
```

Comment:
should this be `componentbaseconfigvalidation`? i'm not quite sure what the rules the community uses but is how many of the objects are imported.

Reply:
Good catch!